python_modules/dagster/dagster/_core/definitions/asset_daemon_cursor.py

```diff
@@ -150,17 +150,17 @@ def with_updates(
                 "Latest storage ID should be >= previous latest storage ID",
             )
 
-        result_latest_evaluation_by_asset_key = {**self.latest_evaluation_by_asset_key}
```

Comment:
This should be part of the underlying PR, right?

Reply:
ah yeah I'll port it over